libs/@local/advanced-types/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0-private",
   "private": true,
   "description": "A collection of helper types",
-  "license": "AGPL-3.0",
+  "license": "MIT",
```

Comment:
In `package.json` when dual-licensing do we always only declare MIT? @Alfred-Mountfield @TimDiekmann

Reply:
IIRC in Crates we do both? Does npm not support this?